docs/source/gravitysensors/tilt.rst

```diff
@@ -1,7 +1,76 @@
+
+Assign Device
+~~~~~~~~~~~~~
+See `Integrating with BrewPi`_
+
+Logging/Visualization
+---------------------
+
+To start logging data points from your Tilt, click below the device name in the orange box. Enter a name for your log in the text field and click "Create Log & Start Logging". This will create a blank graph where your data points will begin to populate after a few moments of being collected. The page does not refresh automatically so you will have to manually refresh to see updates.
+
+Add Data Point
+~~~~~~~~~~~~~~
+
+This button allows you to manually add a data point to the graph. What you enter here will appear on the graph at the time you enter it.
+
+.. todo:: Where does 'extra data' get displayed?
+
+Control Logging
+~~~~~~~~~~~~~~~
+
+This button will give you the option to stop logging data and return you to the dashboard.
+
+Guided Calibration
+------------------
+
+Although Tilt devices seem to output specific gravity readings, they work by internally measuring the angle (tilt) of the device and applying an equation to convert this angle to the expected specific gravity. Although we do not have the ability to edit or calibrate this conversion equation directly, we can apply a correction factor to the readings it provides to add additional accuracy to the specific gravity we log.
+
+This process will guide you through creating sugar/water mixtures of known specific gravity and then seeing what specific gravity the Tilt reports from each. By the end of this process you will have collected 5 data points which Fermentrack will then use to derive your Tilt's conversion function. You will need a few pieces of equipment to complete this process:
+
+* 4 lbs (2 kg) White Table Sugar
+* Water
+* 2 cup (500 ml) measuring cup (for mixing)
+* 2+ gallon (or 6+ liter) bucket (does not need to be sanitized)
+* Tilt Hydrometer
+* Gram scale
+
+Once you have assembled these supplies, click "Begin guided calibration".
+
+Follow the steps in the wizard. You will be creating a liquid with varying levels of known gravities, and then comparing it to the actual measurements reported by the Tilt sensor. This data will be used to mathematically determine a coefficient to apply to your Tilt which result in the most accurate reading possible for future use.
+
+Integrating with BrewPi
+-----------------------
+To integrate your Tilt logging with a BrewPi controller:
+
+#. Select the "Assign Device" button from the Dashboard view
+#. From the dropdown list, select the name of the controller you wish to associate the Tilt sensor with
+#. Click "Attach sensor to controller"
+
+Now, to view your Tilt sensor readings, navigate to the BrewPi controller dashboard. You will have to restart a log if you had one running before associating the Tilt with the BrewPi sensor. Now, any wort you ferment with this controller will incorporate the Tilt's temperature and gravity readings onto your graph.
```

Comment:
One other thing I would note here is the fact that once a gravity sensor is attached to a BrewPi temp controller, the temp controller becomes the primary device driving the operation of the gravity sensor. Although gravity sensor-specific settings (such as the calibration equation) can be edited by going to the gravity sensor's dashboard, logging controls are synchronized with and controlled from the temperature controller's page.